admin_domains.php

```diff
@@ -301,6 +301,20 @@
 
 			$add_date = date('Y-m-d');
 
+			if( (int)\Froxlor\Settings::Get('dkim.dkim_keylength') == 1024){
```

Comment:
why not just use `(int)\Froxlor\Settings::Get('dkim.dkim_keylength')` in the `makeotion` calls - makes it more complicated than necessary

Reply:
I do not have the time to dig into this...